content/en/docs/setup/platform-setup/kubesphere/index.md

```diff
@@ -0,0 +1,63 @@
+---
+title: KubeSphere Container Platform
+description: Instructions to setup a KubeSphere Container Platform for Istio.
+weight: 18
+skip_seealso: true
+keywords: [platform-setup,kubesphere,kubernetes]
+---
+
+Follow these instructions to prepare the [KubeSphere Container Platform](https://github.com/kubesphere/kubesphere) for Istio. You can download KubeSphere to easily install a Kubernetes cluster to your Linux machines.
+
+{{< tip >}}
+KubeSphere provides [All-in-One](https://kubesphere.io/docs/v2.1/en/installation/all-in-one/) and [Multi-Node](https://kubesphere.io/docs/v2.1/en/installation/multi-node/) installation. This enables quick setup and manages Kubernetes and Istio in a unified web console. This tutorial will only walk you through the All-in-One. Reference [Multi-node Installation](https://kubesphere.io/docs/v2.1/en/installation/multi-node/) for further information.
+{{< /tip >}}
+
+## Prerequisites
+
+A new Linux machine (64 bit), at least 2 CPUs and 4GB RAM, supports virtual machine or bare metal. The following operating systems are supported:
```

Comment:
Look back at @sdake's comments. I see a few haven't been acted on such as bullet-izing the pre-reqs and splitting some of the commands.